docs/Developers/Newcomers_Guide.rst

```diff
@@ -97,104 +104,97 @@ Step 2. Grab an Invitation to the Organization
 ----------------------------------------------
 
 Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
+invitation to our org.
+This is your first step towards contributing.
+A maintainer will command ``cobot`` (our gitter bot) to invite you and be part
+of the Newcomer team.
```

Comment:
uhmm... urgh... I get why you want that buuuuut that would mean a lot of manual work on my end :/ Kinda feels like rewriting everything could be easier than separating the changes :(